python/cudf/dataframe/dataframe.py

```diff
@@ -44,74 +44,52 @@ class DataFrame(object):
 
     Build dataframe with `__setitem__`:
 
-    .. code-block:: python
-
-          from cudf import DataFrame
-          df = DataFrame()
-          df['key'] = [0, 1, 2, 3, 4]
-          df['val'] = [float(i + 10) for i in range(5)]  # insert column
-          print(df)
-
-    Output:
-
-    .. code-block:: python
-
-              key  val
-          0    0 10.0
-          1    1 11.0
-          2    2 12.0
-          3    3 13.0
-          4    4 14.0
+    >>> import cudf
+    >>> df = cudf.DataFrame()
+    >>> df['key'] = [0, 1, 2, 3, 4]
+    >>> df['val'] = [float(i + 10) for i in range(5)]  # insert column
+    >>> print(df)
+       key   val
+    0    0  10.0
+    1    1  11.0
+    2    2  12.0
+    3    3  13.0
+    4    4  14.0
 
     Build dataframe with initializer:
 
-    .. code-block:: python
-
-          from cudf import DataFrame
-          import numpy as np
-          import datetime as dt
-          ids = np.arange(5)
-
-          # Create some datetime data
-          t0 = dt.datetime.strptime('2018-10-07 12:00:00', '%Y-%m-%d %H:%M:%S')
-          datetimes = [(t0+ dt.timedelta(seconds=x)) for x in range(5)]
-          dts = np.array(datetimes, dtype='datetime64')
-
-          # Create the GPU DataFrame
-          df = DataFrame([('id', ids), ('datetimes', dts)])
-          print(df)
-
-    Output:
-
-    .. code-block:: python
-
-              id               datetimes
-          0    0 2018-10-07T12:00:00.000
-          1    1 2018-10-07T12:00:01.000
-          2    2 2018-10-07T12:00:02.000
-          3    3 2018-10-07T12:00:03.000
-          4    4 2018-10-07T12:00:04.000
+    >>> import cudf
+    >>> import numpy as np
+    >>> from datetime import datetime, timedelta
+    >>> ids = np.arange(5)
+
```

Comment:
For a single example, when there are newlines between code, numpydoc renders them as separate code blocks.

This could be my stylistic preference, but removing blank lines so that numpydoc renders the full example in a single code-block would be clearer.